### ISSUE TITLE
Версия 1,0,6 просит подписку.

### ISSUE THREAD

**Issue by 1Deluks** [dedpnd/yaradio-yamusic#84]
Переустанавливал, один черт просит, что я делаю не так?

**Comment by iHastr** [comment]
Да, да, подтверждаю! С июня 2023 при запуске вашего приложения на вкладке Радио пишет "Слушайте потоки с Яндекс Плюсом" и невозможно ничего послушать, Этот рекламный баннер загораживает собой интерфейс и его никак не закрыть.

**Comment by MaximKiselev** [comment]
@1Deluks , @iHastr 
кликните на банер мышью и нажмите ESC на клавиатуре

**Comment by iHastr** [comment]
> @1Deluks , @iHastr кликните на банер мышью и нажмите ESC на клавиатуре

Увы, это скроет лишь баннер, а вот вкладка "Потоки" всё равно требует подписку Яндекс Плюс. Именно там и выбирается нужный жанр музыки. Раньше ведь работало без подписки, так что пардон муа))
